CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contribute to TimerX
+
+TimerX is written in **Python**, so you have to install it if you want to contribute.
+You can download  the latest version [here](https://www.python.org/downloads/release/python-3101/).
```

Comment:
There's even Anaconda3 which adds Python to your PATH, does the work for you. Moreover, old versions of Python mean it's easier to manage, right?

Reply:
dunno, i would leave it as it is